### PR TITLE
Elly Han - CS6

### PR DESCRIPTION
**Pull request by ellysalley**

### CODE REVIEW

File: DeploymentExperiences.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Summarizing how the process went for you, what went well and what was tricky, and how far you got
+
+1. Install new dependencies 
+    - Even though I installed all the dependencies, VS code pilint showed me 'unablie to import decouple'. I don't know how to fix this lint error.
```
Comment (agallant) [inline DeploymentExperiences.md:4]: Where you able to get this fixed? Chances are you either need to `pipenv install` or `pipenv shell`.

**Review by agallant — COMMENTED** [review]
Good job! If you got that issue resolved then that's great, but if you still have questions let me know. Thanks!

File: Project.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* Djorg project repo:
+https://github.com/ellysalley/djorg
+
+* Heroku link
```
Comment (TheDeterminator) [inline Project.md:4]: Thanks for putting your link to your Heroku and to your Djorg repo, I will review and make comments briefly there and more in-depth in your sprint review form.

**Review by TheDeterminator — COMMENTED** [review]
Elly, good job figuring out these challenges. Please see my comments in your sprint review for more in-depth feedback

**Pull request closed without merging**